acceptance/tests/connect/connect_proxy_lifecycle_test.go

```diff
@@ -205,3 +205,167 @@ func TestConnectInject_ProxyLifecycleShutdown(t *testing.T) {
 		})
 	}
 }
+
+func TestConnectInject_ProxyLifecycleShutdownJob(t *testing.T) {
+	cfg := suite.Config()
+
+	defaultGracePeriod := 10
+
+	cases := map[string]int{
+		"../fixtures/cases/job-client-inject":                  defaultGracePeriod,
+		"../fixtures/cases/job-client-inject-grace-period-0s":  0,
+		"../fixtures/cases/job-client-inject-grace-period-30s": 30,
```

Comment:
I think 30 seconds is much too long of a test. I'd do 10 as the max test, and 5 as the default.